stl/inc/yvals_core.h

```diff
@@ -799,7 +800,19 @@
 #define _DEPRECATE_STDEXT_HASH_UPPER_BOUND
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4024
+// P0767R1 [depr.meta.types]
+#if _HAS_CXX20 && !defined(_SILENCE_CXX20_IS_POD_DEPRECATION_WARNING) \
+    && !defined(_SILENCE_ALL_CXX20_DEPRECATION_WARNINGS)
+#define _CXX20_DEPRECATE_IS_POD                                              \
+    [[deprecated("warning STL4024: "                                         \
+                 "std::is_pod and std::is_pod_v are deprecated in C++20. "   \

```

Comment:
We need another sentence here to steer people towards the library components they are intended to use instead. How about "The is_trivially_copyable and/or is_standard_layout traits likely suit your use case."?